helpers/trezor.js

```diff
@@ -1,8 +1,33 @@
 import * as wallet from "./wallet";
 import { rawHashToHex, rawToHex } from "./bytes";
+import { networks, address } from "@trezor/utxo-lib";
+
+
+const Buffer = require('safe-buffer').Buffer
 
 const hardeningConstant = 0x80000000;
 const cointype = 1; // 0 = bitcoin, 42 = decred, 1 = decred testnet
+const testnetTicketPoolSize = 1024
+const testnetSubsidyReductionInterval = 2048
+const testnetWorkRewardProportion = 6
+const testnetStakeRewardProportion = 3
+const testnetBlockTaxProportion = 1
+const testnetTicketsPerBlock = 5
+const testnetMulSubsidy = 100
+const testnetDivSubsidy = 101
+const testnetBaseSubsidy = 2500000000
+const simnetTicketPoolSize = 64
+const simnetSubsidyReductionInterval = 128
+const simnetWorkRewardProportion = 6
+const simnetStakeRewardProportion = 3
+const simnetBlockTaxProportion = 1
+const simnetTicketsPerBlock = 5
+const simnetMulSubsidy = 100
+const simnetDivSubsidy = 101
+const simnetBaseSubsidy = 50000000000
```

Comment:
These should be something resembling dcrd's `chaincfg.Params` structure, such that you can easily switch the network by just initializing a different params object.

Reply:
Yeah, sorry. Just was quick to copy paste. Maybe won't need after having dcrwallet do the tx construction work.